### PR TITLE
WinSCP.WinSCP version 5.19

### PR DESCRIPTION
**Pull request by KaranKad**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/17931)

### CODE REVIEW

File: manifests/w/WinSCP/WinSCP/5.19/WinSCP.WinSCP.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+PackageVersion: 5.19
+MinimumOSVersion: 10.0.0.0
+InstallerType: inno
+InstallerSwitches:
+  Custom: /ALLUSERS
+Installers:
+- Architecture: x86
+  Scope: machine
+  InstallerUrl: https://sourceforge.net/projects/winscp/files/WinSCP/5.19/WinSCP-5.19-Setup.exe/download
+  InstallerSha256: ADC50659B2D55751DB5FA000E51335E576A54F8102BA680B231C3939C6A7AB24
```
Comment (OfficialEsco) [inline manifests/w/WinSCP/WinSCP/5.19/WinSCP.WinSCP.installer.yaml:14]:
```suggestion
Installers:
- Architecture: x86
  InstallerType: inno
  Scope: machine
  InstallerUrl: https://sourceforge.net/projects/winscp/files/WinSCP/5.19/WinSCP-5.19-Setup.exe/download
  InstallerSha256: ADC50659B2D55751DB5FA000E51335E576A54F8102BA680B231C3939C6A7AB24
  InstallerSwitches:
    Custom: /NORESTART /ALLUSERS
- Architecture: x86
  InstallerType: inno
  Scope: user
  InstallerUrl: https://sourceforge.net/projects/winscp/files/WinSCP/5.19/WinSCP-5.19-Setup.exe/download
  InstallerSha256: ADC50659B2D55751DB5FA000E51335E576A54F8102BA680B231C3939C6A7AB24
  InstallerSwitches:
    Custom: /NORESTART /CURRENTUSER
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
Compatibility 👀

**Comment by OfficialEsco** [comment]
```
6/17/2021 9:25:57 PM
Status: Completed
##[error] Installer https://sourceforge.net/projects/winscp/files/WinSCP/5.19/WinSCP-5.19-Setup.exe/download
##[error] Result: Failed
##vso[task.logissue type=error] Error occurred while downloading installer from https://sourceforge.net/projects/winscp/files/WinSCP/5.19/WinSCP-5.19-Setup.exe/download. 0x80072EFD.
Error information: Error occurred while querying URL for headers: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond.
```

dumb bot 🤦‍♂️

File: manifests/w/WinSCP/WinSCP/5.19/WinSCP.WinSCP.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: WinSCP.WinSCP
+PackageVersion: 5.19
+MinimumOSVersion: 10.0.0.0
+Installers:
+- Architecture: x86
+  InstallerType: inno
+  Scope: machine
+  InstallerUrl: https://sourceforge.net/projects/winscp/files/WinSCP/5.19/WinSCP-5.19-Setup.exe/download
```
Comment (ImJoakim) [inline manifests/w/WinSCP/WinSCP/5.19/WinSCP.WinSCP.installer.yaml:11]: This should do it :)
```suggestion
  InstallerUrl: https://nav.dl.sourceforge.net/project/winscp/WinSCP/5.19/WinSCP-5.19-Setup.exe

```

File: manifests/w/WinSCP/WinSCP/5.19/WinSCP.WinSCP.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  InstallerSha256: ADC50659B2D55751DB5FA000E51335E576A54F8102BA680B231C3939C6A7AB24
+  InstallerSwitches:
+    Custom: /NORESTART /ALLUSERS
+- Architecture: x86
+  InstallerType: inno
+  Scope: user
+  InstallerUrl: https://sourceforge.net/projects/winscp/files/WinSCP/5.19/WinSCP-5.19-Setup.exe/download
```
Comment (ImJoakim) [inline manifests/w/WinSCP/WinSCP/5.19/WinSCP.WinSCP.installer.yaml:18]:
```suggestion
  InstallerUrl: https://nav.dl.sourceforge.net/project/winscp/WinSCP/5.19/WinSCP-5.19-Setup.exe

```

File: manifests/w/WinSCP/WinSCP/5.19/WinSCP.WinSCP.installer.yaml (hunk 2 above)
Comment (OfficialEsco) [reply manifests/w/WinSCP/WinSCP/5.19/WinSCP.WinSCP.installer.yaml:11]: Hmm, we've been using the first sourceforge link without issues before. Is that a permanent dynamic download center link?

Comment (OfficialEsco) [reply manifests/w/WinSCP/WinSCP/5.19/WinSCP.WinSCP.installer.yaml:11]: I think the Original link is best to use
image

**Comment by OfficialEsco** [comment]
@karankad Try now since another PR got trough

**Review by OfficialEsco — APPROVED** [review]
🎉

**Pull request merged**